algonaut_crypto/src/lib.rs

```diff
@@ -36,6 +38,12 @@ impl<'de> Deserialize<'de> for HashDigest {
     }
 }
 
+impl fmt::Debug for HashDigest {
```

Comment:
Very minor doubt: why don't we use `Debug` instead of `fmt::Debug`? For `fmt::Result` makes sense because of the many clashes.